ci/prow/testgrid.yaml

```diff
@@ -56,6 +56,10 @@ default_dashboard_tab:
   alert_options:
     alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
 test_groups:
+- name: ci-knative-serving-continuous
+  gcs_prefix: knative-prow/logs/ci-knative-serving-continuous
```

Comment:
this doesn't look right, you may want to ask @chizhg

Reply:
The current testgrid config generator does not support such use case.
And it does not sound right to me to add a continuous job that uses Go 1.14, as once the job fails on Go 1.14, there is no point to run it again and again since it will always fail.

Reply:
I'll create a separate PR for the CI then. 

Why is the case that a continuous job will keep failing on Go 1.14? I think after the serving team make the change to use 1.14, wouldn't the CI using go1.14 be more useful in that case? 

The main reason for adding CI currently is to make sure once they support it, if it ever gets broken, it's easier to track down which change break it since the go1.14 version will be optional until that's released everywhere.

Reply:
I reconsidered about this, and I think you are right.

But instead of making these changes, you can probably make them custom jobs, that should make your life easier:
https://github.com/knative/test-infra/blob/77b975af074880c49957946a6d861e063e11f232/ci/prow/config_knative.yaml#L48
https://github.com/knative/test-infra/blob/77b975af074880c49957946a6d861e063e11f232/ci/prow/config_knative.yaml#L355

Reply:
Cool testgrid result looks more reasonable after switcching to custom job :D 
PTAL